cypress/cypress/e2e/course-workflow.cy.ts

```diff
@@ -0,0 +1,99 @@
+describe('Test course creation and editing functionalities', () => {
+  const randomNumber = Math.round(Math.random() * 1000)
+  const name = 'Testkurs ' + randomNumber
+  const displayName = 'Testkurs Display ' + randomNumber
+  const description = 'Lorem ipsum dolor sit amet ... '
+
+  it('Test the creation of a new course', () => {
+    // log into frontend-manage
+    cy.loginLecturer()
+
+    // switch to course list
+    cy.get('[data-cy="courses"]').click()
+
+    // create a new course
+    cy.get('[data-cy="course-list-button-new-course"]').click()
+
+    // fill in the form
+    cy.get('[data-cy="create-course-name"]').type(name)
+    cy.get('[data-cy="create-course-display-name"]').type(displayName)
+    // TODO: Fix the typing into the content input - does not seem to work...
+    cy.get('[data-cy="create-course-description"]').click().type(description)
```

Comment:
For some reason, cypress does not manage to insert a description deterministically. In some cases, parts of the back of the string are typed into the field. However, in most cases, nothing is inserted at all, which also impacts the commented editing part further below.